packaging/fedora/snapd.spec

```diff
@@ -805,6 +831,13 @@ if [ $1 -eq 0 ]; then
 fi
 %endif
 
+%post -n snap-confine
+%if %{with apparmor}
+if [ $1 -eq 1 ] ; then
+    /sbin/apparmor_parser -r /etc/apparmor.d/*snap-confine* > /dev/null 2>&1 || :
+fi
+%endif
```

Comment:
Please wrap the scriptlet section in the conditional, so we don't have an empty scriptlet when it's not in use.

Reply:
Done

Reply:
Do we only need to do this on installation? Or should it happen on upgrades too? If it needs to happen on upgrades as well, then the conditional should be `$1 -ge 1` instead of `$1 -eq 1`.